example_12/controllers/src/passthrough_controller.cpp

```diff
@@ -85,6 +85,11 @@ controller_interface::CallbackReturn PassthroughController::on_configure(
   reference_interfaces_.resize(
     reference_interface_names_.size(), std::numeric_limits<double>::quiet_NaN());
 
+  for (size_t i = 0; i < reference_interface_names_.size(); i++)
+  {
+    REGISTER_ROS2_CONTROL_INTROSPECTION(reference_interface_names_[i], &reference_interfaces_[i]);
```

Comment:
aren't reference interfaces exported by default?

Reply:
Yes yes, they are.
I added to just show how to introspect internal variables. I didn't know what else to register in this controller

Reply:
I see. I had a look now and don't find any other more useful variables to introspect in this repository. maybe we should have a look in ros2_controllers and register more variables to pal_statistics, currently most of the controllers have their own publisher with a custom status message type. not sure what is the best way in the controllers for doing so

Reply:
I would say yes, it is better to register some variables in the controllers